src/spring-cloud/azext_spring_cloud/application_configuration_service.py

```diff
@@ -0,0 +1,209 @@
+
+    repo = _get_existing_repo(acs_resource.properties.settings.git_property.repositories, name)
+
+    acs_resource.properties.settings.git_property.repositories.remove(repo)
+
+    _validate_acs_settings(client, resource_group, service, acs_resource.properties.settings)
+
+    logger.warning("[2/2] Removing item of Application Configuration Service settings, (this operation can take a while to complete)")
+    return sdk_no_wait(no_wait, client.configuration_services.begin_create_or_update, resource_group, service, acs_resource)
+
+
+def application_configuration_service_git_list(cmd, client, service, resource_group):
+    acs_resource = client.configuration_services.get(resource_group, service)
+    acs_settings = acs_resource.properties.settings
+
+    if not acs_settings or not acs_settings.git_property or not acs_settings.git_property.repositories:
+        raise CLIError("Repos not found.")
+
+    return acs_settings.git_property.repositories
+
+
+def application_configuration_service_bind(cmd, client, service, resource_group, app):
+    _acs_bind_or_unbind_app(cmd, client, service, resource_group, app, True)
+
+
+def application_configuration_service_unbind(cmd, client, service, resource_group, app):
+    _acs_bind_or_unbind_app(cmd, client, service, resource_group, app, False)
+
+
+def _acs_bind_or_unbind_app(cmd, client, service, resource_group, app_name, enabled):
+    app = client.apps.get(resource_group, service, app_name)
+    app.properties.addon_configs = {
+        APPLICATION_CONFIGURATION_SERVICE_NAME: models.AddonProfile()
+    } if app.properties.addon_configs is None else app.properties.addon_configs
+
+    if app.properties.addon_configs.get(APPLICATION_CONFIGURATION_SERVICE_NAME).enabled == enabled:
+        logger.warning('App "{}" has been {}binded'.format(app_name, '' if enabled else 'un'))
+        return
+
+    app.properties.addon_configs[APPLICATION_CONFIGURATION_SERVICE_NAME].enabled = enabled
+    return client.apps.begin_update(resource_group, service, app_name, app)
+
+
+def _get_input_repo(repo, patterns, uri, label, search_paths, username, password, host_key, host_key_algorithm, private_key, strict_host_key_checking):
+    if patterns:
+        patterns = patterns.split(",")
+    if search_paths:
+        search_paths = search_paths.split(",")
+
+    repo.patterns = patterns or repo.patterns
+    repo.uri = uri or repo.uri
+    repo.label = label or repo.label
+    repo.search_paths = search_paths or repo.search_paths
+    repo.username = username or repo.username
+    repo.password = password or repo.password
+    repo.host_key = host_key or repo.host_key
+    repo.host_key_algorithm = host_key_algorithm or repo.host_key_algorithm
+    repo.private_key = private_key or repo.private_key
+    repo.strict_host_key_checking = strict_host_key_checking or repo.strict_host_key_checking
+    return repo
+
+
+def _get_existing_repo(repos, name):
+    repos = [r for r in repos if r.name == name]

```

Comment:
`repo = next(r for r in reposeif r.name == name, None)` to find the first or default